site/public/js/ta-grading-rubric.js

```diff
@@ -626,7 +626,7 @@ function ajaxAddComponent(gradeable_id, peer) {
         $.getJSON({
             type: "POST",
             async: AJAX_USE_ASYNC,
-            url: buildCourseUrl(['gradeable', gradeable_id, 'components', 'new']),
+            url: buildCourseUrl(['gradeable', gradeable_id, 'components', peer, 'new']),
```

Comment:
As mentioned above (a few comments ago) peer was removed from the course url here, and instead added to the data object below. You can remove it here.